help/en/html/doc/Technical/XmlView.shtml

```diff
@@ -69,8 +69,8 @@
         "/prefs" link.</li>
 
         <li>
-          <a href="images/Webstartxml.png"><img src="images/Webstartxml.png" height="286" width=
-          "410" alt="web server home page"></a> Finally, click on any needed directory names to get
+          <a href="images/Webstartxml.png"><img src="images/Webstartxml.png"
```

Comment:
I am in process of a significant update to XmlView.shtml.  I was going to put it in a PR tonight but will wait until your PR is accepted so no conflict (although I will incorporate this change in my version as well.

Reply:
Looking forward to have your work on this added, Jerry. Thanks